pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1565,6 +1565,9 @@ self: super: {
   # Upstream issue: https://github.com/haskell-servant/servant-swagger/issues/129
   servant-swagger = dontCheck super.servant-swagger;
 
+  # waiting for aeson bump
```

Comment:
@domenkozar Can you additionally open an issue upstream and link it here? That should make it easier to clean up this override when it becomes unnecessary.